<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="pskindero-ThinkPad-Edge-E545" name="com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest" tests="3" failures="1" timestamp="22 May 2017 19:19:11 GMT" time="0.246" errors="0">
  <testcase name="restGET_AllManagers_ResultOk" time="0.055" classname="com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest"/>
  <testcase name="restGET_AllManagers_Parse" time="0.144" classname="com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest"/>
  <testcase name="restGET_FirstManager_ResultOk" time="0.047" classname="com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest">
    <failure type="java.lang.AssertionError" message="Invalid response status expected [200] but found [204]">
      <![CDATA[java.lang.AssertionError: Invalid response status expected [200] but found [204]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at com.pskindero.javaee.javaeetech.util.BaseRestTest.execGetAndAssertIfNotOkStatus(BaseRestTest.java:22)
	at com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest.restGET_FirstManager_ResultOk(ManagersRestTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- restGET_FirstManager_ResultOk -->
</testsuite> <!-- com.pskindero.javaee.javaeetech.endpoints.ManagersRestTest -->
